cmd/kubeadm/app/apis/kubeadm/v1beta1/conversion.go

```diff
@@ -34,7 +35,8 @@ func Convert_kubeadm_InitConfiguration_To_v1beta1_InitConfiguration(in *kubeadm.
 	return nil
 }
 
-func Convert_kubeadm_JoinControlPlane_To_v1beta1_JoinControlPlane(in *kubeadm.JoinControlPlane, out *JoinControlPlane, s conversion.Scope) error {
+// ConvertKubeadmJoinControlPlaneToV1beta1JoinControlPlane ...
+func ConvertKubeadmJoinControlPlaneToV1beta1JoinControlPlane(in *kubeadm.JoinControlPlane, out *JoinControlPlane, s conversion.Scope) error {
```

Comment:
thanks for the PR, but unless we fix the generator that adds the underscores, this will not work.

we will fix this once this is resolved https://github.com/kubernetes/code-generator/issues/30
note that the same problem applies to other API files under kubernetes/kubernetes.

/close